src/internal_events/file.rs

```diff
@@ -256,6 +257,23 @@ impl InternalEvent for FileCheckpointWriteFailed {
     }
 }
 
+#[derive(Debug)]
+pub struct FileOpen<'a> {
+    pub path: &'a Path,
+    pub since: Instant,
+}
+
+impl<'a> InternalEvent for FileOpen<'a> {
+    fn emit_metrics(&self) {
+        gauge!(
+            "file_open_time_ms", self.since.elapsed().as_millis() as i64,
```

Comment:
```suggestion
            "file_open_since_ms", self.since.elapsed().as_millis() as i64,
```

Better clarity of what time it is

Reply:
`file_open_since_ms` gives me an idea of a moment in ms from which this file has been open, while this metric measures for how long a file has been open, so `file_open_time_ms` fits better.